### ISSUE TITLE
Background calculation error: Unknown action: ~

### ISSUE THREAD

**Issue by tomachalek** [czcorpus/kontext#13]
```
2015-02-22 18:37:31,990 [conclib] ERROR: Background calculation error: Unknown action: ~
2015-02-22 18:37:32,093 [controller] ERROR: Traceback (most recent call last):
  File "/opt/noske/share/kontext/public/../lib/controller.py", line 818, in process_method
    self._invoke_action(method, pos_args, named_args, tpl_data))
  File "/opt/noske/share/kontext/public/../lib/controller.py", line 490, in _invoke_action
    ans = apply(action, args[1:], na)
  File "/opt/noske/share/kontext/public/../lib/actions.py", line 184, in view
    conc = self.call_function(conclib.get_conc, (self._corp(),))
  File "/opt/noske/share/kontext/public/../lib/controller.py", line 513, in call_function
    return apply(func, args, na)
  File "/opt/noske/share/kontext/public/../lib/conclib.py", line 415, in get_conc
    samplesize=samplesize, fullsize=fullsize, minsize=minsize)
  File "/opt/noske/share/kontext/public/../lib/conclib.py", line 362, in _get_async_conc
    raise e
Exception: Unknown action: ~
```

**Comment by tomachalek** [comment]
This is a tough one... First observations:

* most of the requests seem to be from *AhrefsBot/5.0*
* the situation happens if stored query (persistent conc. plugin) has another code as *q* while it should have an actual query there:

E.g. the key *concordance:2gr70MmB* contains:

```json
{
  "align": "", 
  "usesubcorp": "", 
  "q": ["~b1sktnzQ"], 
  "corpname": "susanne", 
  "persist_level": 0, 
  "id": "2gr70MmB"
}
```

**Comment by tomachalek** [comment]
Fixed in rev 9a44fec. Call me a genius.
